### DISCUSSION TITLE
How to play playlist once

### DISCUSSION THREAD

**Discussion by anhdv1979** [AzuraCast/AzuraCast#3881] [Q&A]
How could I stream playlist once and stop ? At the moment it seems that it loop!
Many thanks!

**Comment by mialiaamy** [comment]
Hey everyone,

Great discussion here! I’m really interested in how we can leverage insights from[ top UK PR companies](https://pearllemonpr.com/top-music-pr-companies-uk/) to improve our outreach strategies. It might help us connect better with our audience and increase visibility.

What do you all think? Have any of you worked with PR firms that you’d recommend?

Thanks
